src/cli.js

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env node
```

Comment:
why did you remove this?

Reply:
> After I reordered the plugins, the build:babel script started complaining about the #!/usr/bin/env node line in the cli.js file. I don't know why that's the case :( I think I'll need some help to figure this one out. For now, I just removed it so it can build and test successfully.

Reply:
@luisrudge Putting this back in and running `yarn build:babel` didn't complain for me.

Reply:
@ianwalter in my branch?

Reply:
I'm happy to rollback that if that's the case. Maybe it was an env issue with my machine 🤔

Reply:
@luisrudge Yes, I pulled down your fork.

Reply:
Perfect. I rolled back 😬